File: examples/otp/validate/node/validateOtp.js
const { OtpService } = require("micro-js-client/otp");

// Validate the OTP code
async function validateOtp() {
  let otpService = new OtpService(process.env.MICRO_API_TOKEN);
  let rsp = await otpService.validate({
    code: "656211",
    id: "asim@example.com",
  });
  console.log(rsp);
}

validateOtp();

File: examples/go/otp/validate/validateOtp.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/otp"
)

// Validate the OTP code
func ValidateOtp() {
	otpService := otp.NewOtpService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := otpService.Validate(&otp.ValidateRequest{
		Code: "656211",
		Id:   "asim@example.com",
	})
	fmt.Println(rsp, err)
}
